### ISSUE TITLE
Java 13 support for Skript developers

### ISSUE THREAD

**Issue by Romitou** [SkriptLang/Skript#2869]
### Description
Hello, I can't compile Skript with gradle.
I do not know if it comes from me, in this case please close this issue.
### Steps to Reproduce
I launched the command `gradlew clean build`.
### Errors / Screenshots
https://gist.github.com/Romitou/c6f141f27f946169763b7651b7f903f3
I did not put the full error, only the errors that are displayed, please tell me if I should have put the complete error.

### Server Information
* **Java** :
```
java version "1.8.0_241"
Java(TM) SE Runtime Environment (build 1.8.0_241-b07)
Java HotSpot(TM) Client VM (build 25.241-b07, mixed mode, sharing)
```
* **Skript version:** 2.5-alpha2
* **Gradle:**
```
------------------------------------------------------------
Gradle 5.0
------------------------------------------------------------

Build time:   2018-11-26 11:48:43 UTC
Revision:     7fc****bf2fc5fe0****c8a0f546****bcd987

Kotlin DSL:   1.0.4
Kotlin:       1.3.10
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          13.0.2 (Oracle Corporation 13.0.2+8)
OS:           Windows 10 10.0 amd64
```

Please excuse my English.
I am a beginner, so if the error is due to me, I am sorry and thank you to close this issue.
Have a good day!

**Comment by Romitou** [comment]
Please let me know if I need to give you more information about my problem, which I will be happy to do.

If the cause of the problem is that I did something wrong, please close this issue: I don't want to bother you with a problem that is not yours.

**Comment by Pikachu920** [comment]
Not sure what's going on here but maybe you would find the nightly builds useful? https://github.com/SkriptLang/Skript/actions?query=workflow%3A%22Java+CI%22

**Comment by bensku** [comment]
The full error log is needed in this case.

**Comment by Romitou** [comment]
> The full error log is needed in this case.

Hello, thank you for your answer. I updated with fix #2846, and this gives me a new error.

Build error: https://gist.github.com/Romitou/e96fbefd5e0eabfb8535952bdb2e8772
--debug: https://gist.github.com/Romitou/29057563f374f8611f52a2fe69d134ae

Regarding the original error, I will pass it on to you as soon as possible, sorry.

**Comment by bensku** [comment]
It appears that Gradle version we're using does not support Java 13. We need to update soon; in the mean time, consider using Java 11.

**Comment by Romitou** [comment]
> It appears that Gradle version we're using does not support Java 13. We need to update soon; in the mean time, consider using Java 11.

Thank you very much for your response and your speed.

**Comment by TPGamesNL** [comment]
Fixed in #3768
